src/scenes/ProgressReports/Detail/StatusStepper.tsx

```diff
@@ -39,53 +48,103 @@ export const StatusStepper = ({
   currentIndex =
     currentIndex === Statuses.length - 1 ? currentIndex + 1 : currentIndex;
 
+  const workflowEvents = report?.workflowEvents.filter(
+    (event) => event.status === report.status.value
+  );
```

Comment:
What's the reason for this?

Reply:
We used this to get the last item in the array and used it to display who and when the last change was made at the bottom of the stepper.

Reply:
I don't see the need to filter the events though